internal/pkg/service/stream/storage/model/job.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+)
+
+// Job contains all Keboola-specific data we need for polling jobs.
```

Comment:
If it's keboola specific, why is it here rather than in the keboola bridge?

Reply:
Fixed commentary. It was copied from `keboolasink/bridge/schema/job.go`